### PR TITLE
Improve support of custom ivy home configuration

### PR DESCRIPTION
**Pull request by molekyla**

### CODE REVIEW

File: plugin/src/sbt-test/psv/custom_ivy_home_config/src/main/scala/Demo.scala [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+package com.here.bom
+
+import com.fasterxml.jackson.databind.ObjectMapper
+
+object Demo extends App {
```
Comment (abanias) [inline plugin/src/sbt-test/psv/custom_ivy_home_config/src/main/scala/Demo.scala:23]: Do we run this app somewhere or mention it?

Comment (molekyla) [reply plugin/src/sbt-test/psv/custom_ivy_home_config/src/main/scala/Demo.scala:23]: Actually in this test we do no run this app. But we compile it

Comment (abanias) [reply plugin/src/sbt-test/psv/custom_ivy_home_config/src/main/scala/Demo.scala:23]: I'd name it like, `CustomIvyTest` instead of `Demo`

Comment (molekyla) [reply plugin/src/sbt-test/psv/custom_ivy_home_config/src/main/scala/Demo.scala]: Done

**Review by abanias — APPROVED** [review]

**Pull request merged**